hax/hax/util.py

```diff
@@ -1954,6 +1971,42 @@ def get_process_current_status(self, status_reported: ObjHealth,
                  node, proc_fid, status_current)
         return status_current
 
+    @repeat_if_fails()
+    def get_process_based_node_state(self, node_fid: Fid) -> str:
+        LOG.debug('----xxxxxx inside stopped 4-0')
+        children = self.kv.kv_get(f'm0conf/nodes/{node_fid}/processes',
+                                  recurse=True)
+        search_filter = ['ios', 'confd']
```

Comment:
I think we don't need to have check for specific services. We need to check all the services(including hax).
Also, have you tested this on server pod where there are no ios or confd services?
cc @mssawant

Reply:
Oh, Thanks. I will remove the filters. 

> Also, have you tested this on server pod where there are no ios or confd services?

yet to test. Will post the results.

Reply:
Done, not checking any specific services. All the processes under a node are accounted.